StripeConnect/StripeConnect/Source/EmbeddedComponentManager+Appearance.swift

```diff
@@ -0,0 +1,239 @@
+//
+//  EmbeddedComponentManager+Appearance.swift
+//  StripeConnect
+//
+//  Created by Chris Mays on 8/28/24.
+//
+
+@_spi(STP) import StripeCore
+import UIKit
+
+@_spi(PrivateBetaConnect)
+extension EmbeddedComponentManager {
+    /// Describes the appearance of embedded components.
+    /// - seealso: https://docs.stripe.com/connect/embedded-appearance-options
+    public struct Appearance {
+        var traitCollection: UITraitCollection = .current
```

Comment:
After looking at this again, I guess that `traitCollectionDidChange` is always called before rendering the view, so this is probably less of a concern. But I think it would still be a good idea to move it out of the `Appearance` struct since it's not really part of the Appearance but rather an attribute that determines how to encode the appearance dict.